microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -45,9 +45,9 @@ When a WebView2 is initialized, it will attempt to find a valid runtime on the m
 
 There are several ways to make your WebView2 app use a specified preview channel of Microsoft Edge:
 *  By calling a function.
```

Comment:
Change h2 section heading from "Using code" to "Calling a function" - to remove confusion/inconsistency between the overview list of approaches vs. the section headings.